internal/cli/internal/command/open/open.go

```diff
@@ -0,0 +1,66 @@
+package open
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/skratchdot/open-golang/open"
+	"github.com/spf13/cobra"
+	"github.com/superfly/flyctl/internal/buildinfo"
+	"github.com/superfly/flyctl/internal/cli/internal/app"
+	"github.com/superfly/flyctl/internal/cli/internal/command"
+	"github.com/superfly/flyctl/internal/cli/internal/flag"
+	"github.com/superfly/flyctl/internal/client"
+)
+
+func New() *cobra.Command {
+	const (
+		long = `Open browser to current deployed application. If an optional path is specified, this is appended to the
+		URL for deployed application
+`
+		short = "Open browser to current deployed application"
+
+		usage = "open [PATH]"
+	)
+
+	cmd := command.New(usage, short, long, run, command.RequireSession, command.RequireAppName)
+
+	flag.Add(cmd,
+		flag.App(),
+		flag.AppConfig(),
+	)
+
+	return cmd
+}
+
+func run(ctx context.Context) (err error) {
+	var (
+		path = "/"
+
+		appName = app.NameFromContext(ctx)
+	)
+
+	if flag.Len(ctx) > 1 {
+		return fmt.Errorf("too many arguments - only one path argument allowed")
```

Comment:
why `fmt.Errorf` instead of `errors.New`? additionally, why don't you just set the maximum amount of accepted commands (NArgs) in `New`?